brainscore_core/plugin_management/test_plugins.py

```diff
@@ -108,21 +108,23 @@ def run_all_tests(root_directory: Path, results: Dict):
                 plugin_test_runner()
 
 
-def run_args(root_directory: Union[Path, str], test_file: Union[None, str] = None, test: Union[None, str] = None):
+def run_args(root_directory: Union[Path, str], test_files: Union[None, List[str]] = None, test: Union[None, str] = None):
     """
     Run single specified test or all tests for each plugin.
 
     :param root_directory: the directory containing all plugin types, e.g. `/local/brain-score_language/`
-    :param test_file: path of target test file (optional)
+    :param test_files: List of paths of target test files. If this is `None`, run all tests
     :param test: name of test to run (optional)
     """
     results = {}
-    if not test_file:
+    if not test_files:
         run_all_tests(root_directory=Path(root_directory), results=results)
-    elif test_file and Path(test_file).exists():
-        run_specified_tests(root_directory=Path(root_directory), test_file=test_file, results=results, test=test)
     else:
-        warnings.warn("Test file not found.")
+        for test_file in test_files:
+            if Path(test_file).exists():
+                run_specified_tests(root_directory=Path(root_directory), test_file=test_file, results=results, test=test)
+            else:
+                warnings.warn(f"Test file {test_file} not found.")
```

Comment:
not necessarily part of this PR, so we can also discuss separately, but shouldn't we start screaming when we're supposed to run a test file that doesn't exist?
```suggestion
            assert Path(test_file).exists()
            run_specified_tests(root_directory=Path(root_directory), test_file=test_file, results=results, test=test)
```

Reply:
Not sure what you mean. This is the place where the error is raised if someone calls `run_args()` with test files that don't exist. Is there more screaming you'd like to happen?